### ISSUE TITLE
undefined method `show_deprecation_message!' for AutoprefixerRails:Module

### ISSUE THREAD

**Issue by noraj** [ai/autoprefixer-rails#174]
I have this error in my CI pipeline since commit https://github.com/ai/autoprefixer-rails/commit/43393d87275627e44f282915e0c61fffdfa8dc03 from PR #173 introduced by v9.8.6.2.

```
undefined method `show_deprecation_message!' for AutoprefixerRails:Module
       error  public/stylesheets/site.css
undefined method `show_deprecation_message!' for AutoprefixerRails:Module
/builds/noraj/cv_alexandre_zanni/vendor/ruby/2.7.0/gems/autoprefixer-rails-9.8.6.2/lib/autoprefixer-rails/processor.rb:23:in `process'
/builds/noraj/cv_alexandre_zanni/vendor/ruby/2.7.0/gems/middleman-autoprefixer-2.10.1/lib/middleman-autoprefixer/extension.rb:83:in `prefix'
/builds/noraj/cv_alexandre_zanni/vendor/ruby/2.7.0/gems/middleman-autoprefixer-2.10.1/lib/middleman-autoprefixer/extension.rb:74:in `process'
/builds/noraj/cv_alexandre_zanni/vendor/ruby/2.7.0/gems/middleman-autoprefixer-2.10.1/lib/middleman-autoprefixer/extension.rb:60:in `call'
```

**Comment by mknadler** [comment]
@noraj Don't know if this helps you or not, but I had the same issue. Had just created a project with the same gemfile days earlier and it was working, created one today and had the same error. Diffing the two gemfiles showed that the working one had autoprefixer-rails at `9.8.6.1`, new one with this missing show_deprecation_method had it at `9.8.6.2`.

So, I set `gem 'autoprefixer-rails', '9.8.6.1'` in my Gemfile, ran `bundle update autoprefixer-rails`, and all's well.

**Comment by noraj** [comment]
@mknadler Yeah I know this workaround the problem was introduced by 9.8.6.2 so forcing 9.8.6.1 for example works. But in my case autoprefixer-rails is used by middleman-autoprefixer a sub-system of middleman. That's why I opened https://github.com/middleman/middleman-autoprefixer/issues/38. I can't do a thing until either this repo is fixed or middleman introduce a workaround.

**Comment by ai** [comment]
Fixed by https://github.com/ai/autoprefixer-rails/pull/175

Will be released in 9.8.6.3 soon
